powershell/task_2.ps1

```diff
@@ -0,0 +1,73 @@
+﻿function Validate-WindowsFeatureName {
+    param (
+        [string]$FeatureName
+    )
+    
+    $features = Get-WindowsFeature | Select-Object -ExpandProperty Name
+    return ($features -contains $FeatureName) 
+}
+
+function Check-WindowsFeatureIfInstalled {
+    param (
+        [string]$FeatureName
+    )
+    
+    return ((Get-WindowsFeature $FeatureName).installstate -eq "Installed")
```

Comment:
This will give you different return object types depending on the outcome of the Get-WindowsFeature. 
What if it is $null?